dags/multipod/maxtext_trillium_configs_perf.py

```diff
@@ -51,7 +51,7 @@
               dataset_project=Project.CLOUD_ML_AUTO_SOLUTIONS.value,
               composer_project=Project.CLOUD_ML_AUTO_SOLUTIONS.value,
               dataset_name=metric_config.DatasetOption.XLML_DATASET,
-              cluster=XpkClusters.TPU_V6E_256_MLPERF_CLUSTER,
+              cluster=XpkClusters.TPU_V6E_256_CLUSTER,
```

Comment:
You'll need to change the project and zone for this cluster config

Reply:
fixed!